LEGO1/mxioinfo.cpp

```diff
@@ -3,7 +3,9 @@
 // OFFSET: LEGO1 0x100cc800
 MXIOINFO::MXIOINFO()
 {
-  memset(&m_info, 0, sizeof(MMIOINFO));
+  // This is not good practice, but shouldn't damage anything
+  // because MXIOINFO has no vtable.
+  memset(this, 0, sizeof(*this));
```

Comment:
#59 adds the `DECOMP_STATIC_ASSERT` macro. If we use this to verify the size of `MXIOINFO` equals the `MMIOINFO` struct I think this is acceptable.